public/editor/scripts/ui/menus.js

```diff
@@ -126,7 +126,19 @@ function setupOptionsMenu(bramble) {
     "#editor-pane-nav-options-menu"
   );
 
-  // Font size
+  // Display Font size
+  $("#editor-pane-nav-font-size").ready(function() {
```

Comment:
I don't think this is needed (i.e., the `ready` handler).  Instead, let's just extract the function you have on 136-139 (i.e., the callback) and make it a function we can also call on startup here:

```js
// Whenever the font size changes in Brackets, update what we display in Thimble.
function updateFontSize() {
  var fontSize = bramble.getFontSize();
  $("#editor-pane-nav-font-size").text(fontSize);
}

bramble.on("fontSizeChange", updateFontSize);

// Set the initial font size on startup
updateFontSize();
```

Reply:
ok. Thank you!